### DISCUSSION TITLE
https://docsaid.org/blog/customized-docusaurus-sidebars-auto-count/

### DISCUSSION THREAD

**Discussion by giscus** [DocsaidLab/website#10] [Announcements]
# https://docsaid.org/blog/customized-docusaurus-sidebars-auto-count/

幫 Sidebar 加點新功能。

https://docsaid.org/blog/customized-docusaurus-sidebars-auto-count/

**Comment by frankelinli** [comment]
非常实用的功能。估计官方以后也会增加。
